workflows/reference/gatk-reference.wdl

```diff
@@ -43,51 +48,63 @@ workflow gatk_reference {
         String dbSNP_vcf_url
         #@ except: SnakeCase
         String dbSNP_vcf_name
+        Int known_vcf_disk_size_gb
+        Int reference_fa_disk_size_gb
+        #@ except: SnakeCase
+        Int dbSNP_vcf_disk_size_gb
         #@ except: SnakeCase
         String? dbSNP_vcf_index_url
         #@ except: SnakeCase
         String? dbSNP_vcf_index_name
         String? interval_list_url
         String? interval_list_name
+        #@ except: SnakeCase
+        Int? dbSNP_vcf_index_disk_size_gb
+        Int? interval_list_disk_size_gb
     }
 
     call util.download as fasta_download { input:
         url = reference_fa_url,
         outfile_name = reference_fa_name,
-        md5sum = reference_fa_md5
+        md5sum = reference_fa_md5,
+        disk_size_gb = reference_fa_disk_size_gb,
     }
 
     call samtools.faidx { input:
-        fasta = fasta_download.downloaded_file
+        fasta = fasta_download.downloaded_file,
     }
 
     call picard.create_sequence_dictionary { input:
-        fasta = fasta_download.downloaded_file
+        fasta = fasta_download.downloaded_file,
     }
 
     call util.download as dbsnp { input:
         url = dbSNP_vcf_url,
-        outfile_name = dbSNP_vcf_name
+        outfile_name = dbSNP_vcf_name,
+        disk_size_gb = dbSNP_vcf_disk_size_gb,
     }
 
     if (defined(dbSNP_vcf_index_url) && defined(dbSNP_vcf_index_name)) {
         call util.download as dbsnp_index { input:
             url = select_first([dbSNP_vcf_index_url, "undefined"]),
-            outfile_name = select_first([dbSNP_vcf_index_name, "undefined"])
+            outfile_name = select_first([dbSNP_vcf_index_name, "undefined"]),
+            disk_size_gb = select_first([dbSNP_vcf_index_disk_size_gb, 1]),
```

Comment:
I dislike this a lot. If we're introducing workflow-level parameters, let's just make them required with a default.

Reply:
If there is no `dbSNP_vcf_index_url` then this is never called, but the parameter can still be a required with default.